Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:2.7
+ENV PYTHONUNBUFFERED 1
+ENV DATABASE_URL postgresql://takwimu:takwimu@db:5432/takwimu
+
+RUN mkdir /app
+WORKDIR /app
+
+# Installing OS Dependencies
+RUN apt-get update && apt-get upgrade -y
+
+RUN pip install -U pip setuptools
```

Comment:
Upgrade pip with this too.

Reply:
@DavidLemayian pip is being uploaded here

Reply:
@phillipahereza Missed the -U flag. thanks.